### ISSUE TITLE
VSCode debugger cannot load call stack / dart source during exception handling on macOS with Flutter 3.3.9

### ISSUE THREAD

**Issue by point-source** [flutter/flutter#116810]
I am having [this exact same issue again](https://github.com/flutter/flutter/issues/113645) on version 3.3.9 and the last few versions before it. It was fixed momentarily on master (per the linked thread) and possibly for one minor release but as I generally stay current, it broke again pretty quickly.

**Comment by exaby73** [comment]
Hello @point-source. I cannot reproduce this issue. I can go to the definitions of each trace in the stack in VsCode while debugging for MacOS using latest `Master (3.7.0-5.0.pre.26)`. Please provide some detailed steps to reproduce along with the output to `flutter doctor -v`.

**Comment by point-source** [comment]
Yep, you're correct. It is resolved on master. Perhaps I misinterpreted the result of the previous issue as implying that this would get fixed in a followup patch release rather than waiting for the next minor version. It was further confused because I could have sworn it was working in one of the later 3.3.x releases after that thread closed.

Safe to say then that this is still open awaiting a stable minor 3.x.x release that includes the fix? Any idea when this might go stable? Or should I just be running the beta/master as my daily?

**Comment by exaby73** [comment]
Unfortunately, there is nothing to do now than to wait for the commit to be pushed to a stable release. You can try requesting a [cherry pick](https://github.com/flutter/flutter/wiki/Flutter-Cherrypick-Process) but given that this issue is not very severe, and it doesn't affect most people I think, I don't think it would be a valid cherry pick request. But you can try. Don't take my word for it ;)
